package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@adobe/aio-lib-templates",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

it is not a bug fix, let's use `2.1.0`
```suggestion
  "version": "2.1.0",
```